js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
```

Comment:
I know we tentatively decided to put this function in app, but after some thought, I think this would be better in it's own utility module (be it some "follow" specific one or a more generic "request" one where other non-model / collection server calls can go).

App was mainly designed to be something to throw instances on that are needed app wide (e.g. the router), more or less a glorified 'window'. I want to avoid it becoming what it turned into in v1, where it seemed to become a dumping ground for anything that could be needed in many parts of the app. Most of that stuff would be better served in a more semantically named utility module.

Another reason it's dangerous to put stuff in app is because app is used so prevalently that it's easy to get into circular dependency issues.

I know it's a very blurry line on what should go in here and I'm not always 100% clear myself, but I tend to err on keeping it very lean.

Just my two cents.